model/host/host.go

```diff
@@ -1515,6 +1515,18 @@ func (h *Host) IsIdleParent() (bool, error) {
 	return num == 0, nil
 }
 
+func (h *Host) UpdateParentIDs() error {
```

Comment:
nit: This should only update one host, right? Shouldn't this be `UpdateParentID`?

Reply:
Nope, it updates all containers on this parent host